version_history.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 2.6.6
+- Added the ability to launch Marbleland levels directly from URLs
+- Added Marbleland leaderboard integration API endpoint.
```

Comment:
I think these are both quite technical and won't make sense to most users. Maybe something like "Improved integration with Marbleland" should suffice here?